service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -119,7 +111,6 @@ void shouldSealPDFAndAddToSealedCMOsListWhenJudgeApprovesOrder() throws Exceptio
     @Test
     void shouldUpdateStateToIssueResolutionWhenNextHearingTypeIsIssueResolutionAndCmoDecisionIsSendToAllParties()
```

Comment:
these 2 test are identical except HearingType, can be parametrised

Reply:
Discussed to leave.